CHANGELOG.md

```diff
@@ -66,7 +66,7 @@ The new architecture helped to introduce new features without performance and co
 
 ### Breaking Changes
 
-- Data schema outputted by romcal has been drastically remodeled. Please have look to the documentation for more informations.
+- Data schema outputted by romcal has been drastically remodeled. Please have look to the documentation for more information.
```

Comment:
I prefer using `output` in the past tense, although I know (but don’t use it) that `outputted` is grammatically correct word. We might consider changing this too.

```suggestion
- Data schema output by romcal has been drastically remodeled. Please have look to the documentation for more information.
```